__tests__/git.spec.js

```diff
@@ -275,4 +276,104 @@ describe('Git', function() {
 		);
   });
 
+  describe('RevList', function() {
+
+    it('requires at least 1 argument', function() {
+      runCommand('git rev-list', function(commandMsg) {
+        expect(commandMsg).toContain('at least 1');
+      });
+    });
+
+    describe('supports', function() {
+      var SETUP = 'git co -b left C0; gc; git merge master; git co -b right C0; gc; git merge master; git co -b all left; git merge right; ';
+
+      it('single included revision', function() {
+        runCommand(SETUP + 'git rev-list all', function(commandMsg) {
+          expect(commandMsg).toBe('C6\nC5\nC4\nC3\nC2\nC1\nC0\n');
+        });
+      });
+
+      it('single excluded revision', function() {
+        runCommand(SETUP + 'git rev-list all ^right', function(commandMsg) {
+          expect(commandMsg).toBe('C6\nC3\nC2\n');
+        });
+      });
+
+      it('multiple included revisions', function() {
+        runCommand(SETUP + 'git rev-list right left', function(commandMsg) {
+          expect(commandMsg).toBe('C5\nC4\nC3\nC2\nC1\nC0\n');
+        });
+      });
+
+      it('multiple excluded revisions', function() {
+        runCommand(SETUP + 'git rev-list all ^right ^left', function(commandMsg) {
+          expect(commandMsg).toBe('C6\n');
+        });
+      });
+    });
+  });
+
+  describe('Log supports', function() {
+    var SETUP = 'git co -b left C0; gc; git merge master; git co -b right C0; gc; git merge master; git co -b all left; git merge right; ';
```

Comment:
sometimes i use serialized tree states for this type fo setup, but this works as well!

Reply:
Yeah, serialized tree state would probably be more in keeping with the way the other tests are setup. I'll be perfectly honest: just entering the commands was easier :)